src/net_processing.h

```diff
@@ -94,6 +94,9 @@ class PeerLogicValidation final : public CValidationInterface, public NetEventsI
 private:
     int64_t m_stale_tip_check_time; //!< Next time to check for stale tip
 
+    /** Whether we've completed initial sync yet, for determining when to turn
```

Comment:
You can mention `CanDirectFetch` as the completed-initial-sync-yet condition.

Reply:
I don't think that's necessary here.